tensor2tensor/data_generators/text_encoder.py

```diff
@@ -113,18 +124,18 @@ def __init__(self, vocab_filename, reverse=False, num_reserved_ids=2):
     self._reverse = reverse
     if vocab_filename is not None:
```

Comment:
OK - the if statement came from an earlier fix, see https://github.com/tensorflow/tensor2tensor/commit/2f03f0cad63c2142595aaffdacc72427496653f3